folium/utilities.py

```diff
@@ -305,7 +305,11 @@ def base(x):
     quants = [0, 50, 75, 85, 90]
```

Comment:
Nope.  Just get the line `quants = [0, 50, 75, 85, 90]` on top of `return [base(np.percentile(arr, x)) for x in quants]`.